x-pack/plugins/cases/server/client/metrics/get_case_metrics.ts

```diff
@@ -67,31 +64,45 @@ const buildHandlers = (
   params: CaseMetricsParams,
   casesClient: CasesClient,
   clientArgs: CasesClientArgs
-): Map<string, MetricsHandler> => {
-  const handlers = [
+): Set<MetricsHandler> => {
+  const handlers: MetricsHandler[] = [
     new Lifespan(params.caseId, casesClient),
     new AlertsCount(params.caseId, casesClient, clientArgs),
-    new AlertDetails(),
+    new AlertDetails(params.caseId, casesClient, clientArgs),
     new Connectors(),
   ];
 
-  const handlersByFeature = new Map<string, MetricsHandler>();
+  const uniqueFeatures = params.features.reduce((acc, feature) => {
+    acc.add(feature);
+    return acc;
+  }, new Set<string>());
```

Comment:
just curiosity, is there any difference with `const uniqueFeatures = new Set(params.features);`?

Reply:
Great idea, totally forgot you could do that!